scala/build/Dockerfile.latest

```
@@ -0,0 +1,148 @@
+# Create a virtual environment
+# Activate the virtual environment
+# Upgrade pip within the virtual environment
+# Copy the requirements file into the container
+# Install dependencies from the requirements file
+# RUN python3 -m venv /venv
+# ENV PATH="/venv/bin:$PATH"
+# RUN pip install --upgrade pip
+# COPY scala/build/requirements.txt .
+# RUN pip install -r requirements.txt
+COPY scala/build/requirements.txt /tmp/requirements.txt
+RUN python3 -m venv /venv && \
```

Hadolint

[info] Not following: File not included in mock.

(SC1091)